sql/core/src/test/resources/sql-tests/inputs/sql-on-files.sql

```diff
@@ -1,19 +1,30 @@
+CREATE DATABASE IF NOT EXISTS sql_on_files;
 -- Parquet
+CREATE TABLE sql_on_files.TEST_PARQUET USING PARQUET AS SELECT 1;
```

Comment:
can we lower-case the table name as well? in case some file system is case sensitive.

Reply:
That makes two of us.